tasks/build_app.js

```diff
@@ -4,15 +4,56 @@ var gulp = require('gulp');
 var less = require('gulp-less');
 var watch = require('gulp-watch');
 var batch = require('gulp-batch');
+var exec = require('child_process').exec;
 var plumber = require('gulp-plumber');
 var jetpack = require('fs-jetpack');
 var bundle = require('./bundle');
 var utils = require('./utils');
+var gutil = require('gulp-util');
 
 var projectDir = jetpack;
 var srcDir = jetpack.cwd('./src');
 var destDir = jetpack.cwd('./app');
 
+gulp.task('dependencies', function () {
+    var rootDir = jetpack.cwd('./').path(),
+        platform = process.platform,
+        buildScript = jetpack.cwd('./scripts').path('build/' + platform + '/build.sh'),
+        buildDir = rootDir + '/build',
+        buildDestDir = rootDir + '/thirdparty';
+
+      return new Promise(function(resolve, reject) {
+          if (!jetpack.exists(buildScript)) {
+            gutil.log("No build file for", gutil.colors.cyan("'" + platform + "'"));
+            return resolve();
+          }
+
+          gutil.log("Going to build for", gutil.colors.cyan("'" + platform + "'"));
```

Comment:
`gutil.log("Going to build for", gutil.colors.cyan("'" + platform + "'"));` should be `gutil.log('Going to build for', "'" +  gutil.colors.cyan(platform) + "'");`